src/AddTodoForm.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function AddTodoForm () {
+
+    return (
+        <form>
+            <label htmlFor='todoTitle'>Title</label>
+            <input id="todoTitle"></input>
```

Comment:
Ok so `<input />` elements are generally self-closing like this:

` <input id="todoTitle" />`

If you want to make the input a bit more descriptive for users, you can also add a `placeholder` like this:

` <input id="todoTitle" placeholder='Enter title here' />`

And to make it so that a value is required, add the `required` attribute:

` <input id="todoTitle" placeholder='Enter title here' required />`

Reply:
Ok, for me was weird because VScode autocompleted for me , so I had my doubts because I couldn't remember if it was self closing tag. But thank you anyways I learn something new today. Always appreciate it your feedback! One of this days I'll set up an 1:1 with you so you can help me to set up my VScode better for the auto closing semi colon and the other things.

Reply:
Yea it'll do that. It's just a thing you kind of have to keep in mind. It doesn't hurt anything having the closing tag, it's just unnecessary and not clean code. Looking forward to it! If my listed times don't work for you just shoot me a message and I'll try and find a time that does.